js/index.js

```diff
@@ -0,0 +1,132 @@
+var url;
+var currency;
+var gbp = document.getElementById("gbp");
+var usd = document.getElementById("usd");
+var btc = document.getElementById("btc");
+var next = document.getElementById("next");
+var back = document.getElementById("back");
+var content = document.getElementById("content");
+
+
+
+usd.addEventListener("click", () => {
+    currency = "USD";
+    url = "https://api.coinranking.com/v1/public/coins?base=USD&timePeriod=24h";
+    var request = new XMLHttpRequest();
+    request.open('GET', url);
+    request.onload = () => {
+        if (request.status >= 200 && request.status < 400) {
+            var data = JSON.parse(request.responseText);
+            var coins = data.data.coins;
+            content.innerHTML = "";
+            for (var i = 0; i < coins.length; i++) {
+                displayCards(coins, currency, i);
+            }
+        } else {
+            window.alert("There's a problem contacting the server, Please refresh or try again in a few moments.");
+        };
+    };
+
+    request.onerror = () => {
+        window.alert("There's a problem contacting the server, Please refresh or try again in a few moments.");
+    }
+    request.send();
+})
+
+gbp.addEventListener("click", () => {
```

Comment:
Instead of repeating code for currency. Write a single function that will be called by each of the button `USD`, `GBP`, etc. Then attach an inline event listener to each of them.  To know which currency wa clicked use `event.target.id` to get the id of the particular button and then make a search for it.

Reply:
Hi Godwin, I made changes to the code but used a dropdown instead of the buttons.